### PR TITLE
Add stars count and last pushed date for GitHub repos

### PR DESCRIPTION
**Pull request by chaconnewu**
I created a separate file that attach repo info (stars count, and last pushed date) for each GitHub repositories included in README.md. It will help users quickly get the basic info of a certain repo.

I can update the file regularly (e.g. daily, weekly) or as needed.

View the file directly at (for format is a little off because of pandoc generated .md):
https://github.com/chaconnewu/awesome-vue/blob/master/awesome_vue_with_repo_info.md

### CODE REVIEW

File: awesome_vue_with_repo_info.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+-   [IntelliJ IDEA / WebStorm](https://github.com/henjue/vue-for-idea) <span> | ★ 134, pushed 27 days ago | </span> by @henjue
+-   [Emacs](https://github.com/CodeFalling/vue-mode) <span> | ★ 2, pushed 8 days ago | </span> by @CodeFalling
+
+### Snippets
+
+-   [vue-snippets](https://atom.io/packages/vue-snippets) for Atom.io by [@ealves-pt](https://github.com/ealves-pt)
+
+### Libraries & Plugins
+
+-   \#\#\#\# Routing
```
Comment (yyx990803) [inline awesome_vue_with_repo_info.md]: these sub headings seem to be incorrectly escaped.

**Comment by yyx990803** [comment]
Thanks, this is cool, but I find the info text a big too big and makes the items harder to scan. Maybe wrap them in `<sup></sup>` instead of `<span>`?

**Comment by chaconnewu** [comment]
Do you mean like this?
https://github.com/chaconnewu/dashboard_for_awesomes/blob/master/awesomes/awesome_vue_with_repo_info.md

But this is a html file. For generated .md file, <sup> and <span> seems make no difference.

**Comment by yyx990803** [comment]
I'm mostly concerned about HTML readability. I think removing the `|` surrounding the info and move it behind the description would be better:

- [vue-cli](#): official CLI for scaffolding Vue.js projects. <sup>★ 561, pushed 9 days ago</sup>

**Comment by chaconnewu** [comment]
Updated. Please take a look at the new commit.

**Comment by azamat-sharapov** [comment]
> I can update the file regularly (e.g. daily, weekly) or as needed.

Probably automate it somehow, to make it "reactive"?

**Comment by chaconnewu** [comment]
It is "half automated" already. I have one script to parse README.md, get data, and attach info. The only thing I do manually is sending PR.

**Comment by azamat-sharapov** [comment]
It feels like this should be hosted as a webpage, maybe on github.io. So that you could dynamically attach those stats..

**Pull request merged**